css/reset.css

```diff
@@ -11,7 +11,7 @@ html{
 }
 
 body{
-    background-color: var(--white-color);
+    background-color: var(--white-color) !important;
```

Comment:
try to avoid !important in css class
use CSS Specificity for apply into element

Reply:
Actually i have imported the styles from component library. so to overwrite the changes i have marked it with important.